src/credentials/CHIPCert.h

```diff
@@ -0,0 +1,460 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2019 Google LLC.
+ *    Copyright (c) 2013-2017 Nest Labs, Inc.
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      This file defines data types and objects for modeling and
+ *      working with CHIP certificates.
+ *
+ */
+
+#pragma once
+
+#include <string.h>
+
+#include <asn1/ASN1.h>
+#include <core/CHIPConfig.h>
+#include <core/CHIPTLV.h>
+#include <crypto/CHIPCryptoPAL.h>
+#include <support/DLLUtil.h>
+
+namespace chip {
+namespace Credentials {
+
+using chip::ASN1::OID;
```

Comment:
we try to avoid `using` in headers

Reply:
+1. I believe even more than try: we got bit by this because header include order starts to matter on name clashes. Please do not have using in headers at all.

Reply:
fixed